lib/active_merchant/billing/gateways/stripe_payment_intents.rb

```diff
@@ -326,6 +327,19 @@ def add_ntid(post, options = {})
         post[:payment_method_options][:card][:mit_exemption][:network_transaction_id] = options[:network_transaction_id] if options[:network_transaction_id]
       end
 
+      def add_claim_without_transaction_id(post, options = {})
+        return if options[:stored_credential] || options[:network_transaction_id] || options[:ds_transaction_id]
```

Comment:
why do we need to back out when these are present?

Reply:
I didn't want to bother with further checks if any transaction ids were present, even if someone passed a `claim_without_transaction_id` field value. But, they may be a bit too opinionated 😅 . If you think that check isn't necessary, I can remove it.

Reply:
Got it, makes sense!